mixer/adapter/stackdriver/stackdriver.go

```diff
@@ -23,7 +23,10 @@ import (
 
 	multierror "github.com/hashicorp/go-multierror"
 
+	md "cloud.google.com/go/compute/metadata"
```

Comment:
nit: run `bin/fmt.sh` on this file to clean up the imports.

Reply:
Done.